document/sections/03a-motivation.adoc

```diff
@@ -0,0 +1,96 @@
+
+ While humans can often resolve spatial questions such as “Is cube A inside cylinder B?” — even when the underlying data comes from different systems like GIS and BIM — this typically relies on implicit understanding, conventions, or direct communication between the data creators. For datasets to be meaningfully aligned, the people who created them often need to clarify assumptions, data structures, coordinate systems, or modeling intentions. Without such human-to-human explanation, machines struggle to interpret spatial relationships across heterogeneous sources. When using machines to analyze geometries and spatial relationships in heterogeneous datasets, a conversion process is required. Data is made homogeneous for analysis through extraction, transformation, and loading. Based on a homogeneous dataset, conclusions can then be drawn. Performing conversions may result in geometric simplifications, which can limit or even eliminate the functionality of spatial queries <<Biljecki2019>>.
+ If one desires digital, automated processing or analysis of 3D spatial questions, spatial information must be structured in an explicitly machine-readable format. When this is done, initiatives based on federated systems with heterogeneous datasets can obtain computer-generated answers to spatial 3D queries without requiring conversion. This highlights the importance of semantic alignment and standardized geometry processing in machine-driven analysis. <<Göbel2024>>
+
+ When performing federated queries on heterogeneous spatial datasets without conversion, a common vocabulary is required to semantically define the 3D object and its 3D spatial function. For both humans and machines, it must be unambiguous what is meant by a query such as:
+
+ “Does the '3D geometry' of the proposed construction '3D-intersect' with the '3D geometry'of an existing asset owned by another party?”
+
+ It should not matter whether the data is provided in a 3D-GIS, 3D-BIM or 3D-CG standard vocabulary.  
+
+=== Added value of 3D in the GeoSPARQL Vocabulary for Stakeholders
+==== We aim to support data Consumers by:
+ - access and understanding of 3D-geometry and 3D-topology from multiple sources; 
+ - the integration of 3D-data geospatial and non-geospatial;
+ - data quality evaluation of the given 3D-data;
+ - Geometric and topological description in natural language.
+
+==== We aim to support Data Providers by:
+ - Maximize the use of 3D-geometry and 3D-topology data;
+ - Defining multiple 3D topology or geometries;
+ - To quality-assure 3D geospatial data.
+
+
+=== Semantic-functional value of 3D-GeoSPARQL
+ Value for data reasoning, formal description, and semantic interoperability
+
+==== Meaningful creation and querying of 3D geometry
+ Whether manually or programmatically defining 3D geometries in BIM, GEO or CG environments, the GeoSPARQL standard could provide or connect to vocabulary to represent 3D structures. A 3D cube, for example, can be represented in multiple ways:
```

Comment:
```suggestion
Building on 3D geometries defined in BIM, GEO or CG, a GeoSPARQL 3D specification would provide precise semantic associations to a vocabulary of 3D structures. A 3D cube, for example, can be represented in multiple ways:
```

Reply:
What does CG stand for? I didn't see the abbreviation explained in earlier text. (could be I'm missing some context)